cfg/lang/russian.cfg

```diff
@@ -368,4 +368,4 @@ help_start       = "Начать комнату"
 help_switch      = "Сменить команду"
 help_unready     = "Стать неготовым"
 help_whisper     = "Отправить сообщение в ЛС"
-help_whois       = "Отобразить информацию об игроке"
+help_whois       = "Отобразить информацию об игроке"
```

Comment:
Can you add a line break at the end of the file? Thanks

Reply:
@Levak, do you mean add a new empty string? Or add a line break before the last line? I don't quite understand, sorry)